utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
+
+	-o <file>
+		Set the output file. Existing files will be overwritten
+		without asking. The file will be an iso image.
+
+	-l <label>
+		Define a custom disk label.
+
+	-r <dir>
+		Set the source rootfs. The rootfs itself will not be changed.
+
+	-t
+		Exclude tarballs directory.
+
+	-d
+		Dry run: Only check config and stuff
+
+	<rootfs>/boot will be copied to the new isofs root and completed by
+	an squashfs containing the rest of the system. The initramfs will be
+	responsible for finding and mounting that squashfs.
+
+	If you use this, prepare to have OVERLAY_FS enabled on your kernel,
+	otherwise your cdrom system will be read-only.
```

Comment:
Yes, but the current KEEP/kernel.config seems to be an bit outdated or missing a lot of directives (OVERLAY_FS is not even marked as "not set", just missing)